controllers/designate_controller.go

```diff
@@ -949,39 +949,6 @@ func (r *DesignateReconciler) apiDeploymentCreateOrUpdate(ctx context.Context, i
 	return deployment, op, err
 }
 
-func (r *DesignateReconciler) centralDeploymentCreateOrUpdate(ctx context.Context, instance *designatev1beta1.Designate) (*designatev1beta1.DesignateCentral, controllerutil.OperationResult, error) {
```

Comment:
Removed because it was failing with below error:

Error:

Running [/home/runner/golangci-lint-1.53.3-linux-amd64/golangci-lint run --out-format=github-actions --timeout 5m] in [] ...
Error: func (*DesignateReconciler).sinkDeploymentCreateOrUpdate is unused (unused)

Error: issues found
Ran golangci-lint in 3459ms